app/jobs/subscription_placement_job.rb

```diff
@@ -28,29 +24,14 @@ def proxy_orders
   end
 
   def place_order_for(proxy_order)
-    JobLogger.logger.info("Placing Order for Proxy Order #{proxy_order.id}")
-    initialise_order(proxy_order)
-    place_order(proxy_order.order)
-  end
+    place_order = PlaceOrder.new(
```

Comment:
Yes, it desperately needs to be renamed. Both seem fine but I might pick `PlaceProxyOrder`.

Reply:
I thought I pushed a commit to change this - did it get lost?

Reply:
Actually maybe it's fine to keep it as PlaceOrder. We're doing it in the context of Subscriptions so that's pretty clear, and we actually are creating the order based on the existing proxy order. So I think the name is fine 👍

Reply:
Ups I may have missed your commit 😅

Reply:
Agree `PlaceProxyOrder` would be an improvement :+1: